assets/js/pages/ClusterDetails/AttributesDetails.jsx

```diff
@@ -1,9 +1,42 @@
 import React, { useState } from 'react';
 
+import { capitalize } from 'lodash';
+
 import Button from '@common/Button';
 import Modal from '@common/Modal';
 import Table from '@common/Table';
 
+export const resourceTableConfig = {
```

Comment:
I'm exporting the config, so it can be reused in tests. So if we add or delete a new key value pair, we don't need to change the tests in AscsErsClusterDetails and HanaClusterDetails. An alternative would be just to hard code the table titles/keys in the test and adjust them every time we change something. 

Any preferences or better options?